lib/presentation/login/view/login_page_view.dart

```diff
@@ -0,0 +1,263 @@
+    return Form(
+      key: _formKey,
+      child: Column(
+        crossAxisAlignment: CrossAxisAlignment.start,
+        children: [
+          SizedBox(
+            height: 10,
+          ),
+          TextFormField(
+            maxLines: 1,
+            enableInteractiveSelection: true,
+            keyboardType: TextInputType.emailAddress,
+            maxLength: 30,
+            decoration: InputDecoration(
+              counterText: "",
+              focusedBorder: OutlineInputBorder(
+                borderRadius: BorderRadius.circular(10),
+                borderSide: BorderSide(color: Colors.black, width: 1),
+              ),
+              alignLabelWithHint: true,
+              labelStyle: TextStyle(color: Colors.black),
+              labelText: "Email",
+              border: OutlineInputBorder(
+                borderRadius: BorderRadius.circular(10),
+                borderSide: BorderSide(
+                  color: Colors.grey,
+                  width: 1,
+                ),
+              ),
+            ),
+            validator: (value) {
+              if (value!.isNotEmpty) {
+                email = value;
+                return null;
+              } else {
+                return "Empty email";
+              }
+            },
+          ),
+          SizedBox(
+            height: 30,
+          ),
+          TextFormField(
+            maxLines: 1,
+            obscureText: !passwordVisible,
+            enableInteractiveSelection: true,
+            keyboardType: TextInputType.text,
+            maxLength: 16,
+            decoration: InputDecoration(
+              counterText: "",
+              focusedBorder: OutlineInputBorder(
+                borderRadius: BorderRadius.circular(10),
+                borderSide: BorderSide(color: Colors.black, width: 1),
+              ),
+              alignLabelWithHint: true,
+              labelStyle: TextStyle(color: Colors.black),
+              labelText: "Password",
+              suffixIcon: IconButton(
+                icon: Icon(
+                  passwordVisible ? Icons.visibility : Icons.visibility_off,
+                  color: Colors.black,
+                ),
+                onPressed: () {
+                  setState(
+                    () {
+                      passwordVisible = !passwordVisible;
+                    },
+                  );
+                },
+              ),
+              border: OutlineInputBorder(
+                borderRadius: BorderRadius.circular(10),
+                borderSide: BorderSide(
+                  color: Colors.grey,
+                  width: 1,
+                ),
+              ),
+            ),
+            validator: (value) {
+              if (value!.isNotEmpty) {
+                password = value;
+                return null;
+              } else {
+                return "Empty password";
+              }
+            },
+          )
+        ],
+      ),
+    );
+  }
+
+  void login() async {
```

Comment:
you should validate form before login() call or you should rename method from login to onLoginPressed